### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OTP-react-redux/OTP-RR Main UI](https://hosted.weblate.org/projects/otp-react-redux/otp-rr-main-ui/).

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by binh-dam-ibigroup — APPROVED** [review]

File: i18n/en-US.yml [hunk 1]
```diff
@@ -44,8 +44,10 @@ actions:
     fetchFieldTripError: "Error fetching field trip: {err}"
     fetchFieldTripsError: "Error fetching field trips: {err}"
     fetchTripsForDateError: "Error fetching trips for field trip travel date: {err}"
-    incompatibleTripDateError: Planned trip date ({tripDate}) is not the requested day of travel ({requestDate})
-    itineraryCapacityError: "Cannot Save Plan: This plan could not be saved due to a lack of capacity on one or more vehicles. Please re-plan your trip."
+    incompatibleTripDateError: Planned trip date ({tripDate}) is not the requested
+      day of travel ({requestDate})
```
Comment (AdrianaCeric) [inline i18n/en-US.yml:48]:
```suggestion
    incompatibleTripDateError: Planned trip date ({tripDate}) is not the requested day of travel ({requestDate})
```
^ Remove large space

File: i18n/en-US.yml [hunk 2]
```diff
@@ -44,8 +44,10 @@ actions:
...
+    itineraryCapacityError: "Cannot Save Plan: This plan could not be saved due to\
+      \ a lack of capacity on one or more vehicles. Please re-plan your trip."
```
Comment (AdrianaCeric) [inline i18n/en-US.yml:50]: What is the need for these slashes?

File: i18n/en-US.yml [hunk 3]
```diff
@@ -67,8 +69,10 @@ actions:
     itineraryExistenceCheckFailed: Error checking whether your selected trip is possible.
     preferencesSaved: Your preferences have been saved.
     smsInvalidCode: The code you entered is invalid. Please try again.
-    smsResendThrottled: A verification SMS was sent to the indicated phone number less than a minute ago. Please try again in a few moments.
-    smsVerificationFailed: Your phone could not be verified. Perhaps the code you entered has expired. Please request a new code and try again.
+    smsResendThrottled: A verification SMS was sent to the indicated phone number
+      less than a minute ago. Please try again in a few moments.
```
Comment (AdrianaCeric) [inline i18n/en-US.yml:73]:
```suggestion
    smsResendThrottled: A verification SMS was sent to the indicated phone number less than a minute ago. Please try again in a few moments.
```
^ Remove large space

File: i18n/en-US.yml [hunk 4]
```diff
@@ -241,14 +246,9 @@ components:
...
-    itinerariesFound: "{itineraryNum, plural,
-        one {# itinerary found}
-        other {# itineraries found}
-      }"
-    numIssues: "{issueNum, plural,
-        one {# issue}
-        other {# issues}
-      }"
+    itinerariesFound: "{itineraryNum, plural, one {# itinerary found} other {# itineraries\
+      \ found} }"
```
Comment (AdrianaCeric) [inline i18n/en-US.yml:250]: See previous question about slashes

File: i18n/en-US.yml [hunk 5]
```diff
@@ -67,8 +69,10 @@ actions:
     itineraryExistenceCheckFailed: Error checking whether your selected trip is possible.
     preferencesSaved: Your preferences have been saved.
     smsInvalidCode: The code you entered is invalid. Please try again.
-    smsResendThrottled: A verification SMS was sent to the indicated phone number less than a minute ago. Please try again in a few moments.
-    smsVerificationFailed: Your phone could not be verified. Perhaps the code you entered has expired. Please request a new code and try again.
+    smsResendThrottled: A verification SMS was sent to the indicated phone number
+      less than a minute ago. Please try again in a few moments.
+    smsVerificationFailed: Your phone could not be verified. Perhaps the code you
+      entered has expired. Please request a new code and try again.
```
Comment (AdrianaCeric) [inline i18n/en-US.yml:75]: See above suggestion on large spaces ( which also applies to other changes in this PR)

**Review by AdrianaCeric — APPROVED** [review]
LGTM! Excited for these new languages. A few formatting comments

File: i18n/en-US.yml (hunk 1 above)
Comment (binh-dam-ibigroup) [reply i18n/en-US.yml:48]: This was auto-formatted by Hosted Weblate, so I'm going to reject.

File: i18n/en-US.yml (hunk 2 above)
Comment (binh-dam-ibigroup) [reply i18n/en-US.yml:50]: This was auto-formatted by Weblate.

File: i18n/en-US.yml (hunk 3 above)
Comment (binh-dam-ibigroup) [reply i18n/en-US.yml:73]: This was auto-formatted by Weblate.

**Pull request merged**